proposal/matplotlib_cartopy.tex

```diff
@@ -177,7 +181,7 @@
 %% Finally, we get to the document.
 \begin{document}
 \title{Improving the Foundations and Maintenance of Matplotlib}
-\author{Dr. Thomas A Caswell}
+\author{Thomas A Caswell}
```

Comment:
Feel free to put this back in, but in my field no one would include "Dr" in an Author list, and it might cause some head scratching.

Reply:
That would also be my instinct, but the reference proposal I have been looking at uses "Dr." in a bunch of places